src/core/server/saved_objects/import/check_conflict_for_data_source.test.ts

```diff
@@ -318,6 +318,82 @@ describe('#checkConflictsForDataSource', () => {
     );
   });
 
+  /*
+   * Timeline test cases
```

Comment:
Can you add a test case where there are multiple `.opensearch()` queries but one is a local cluster and one is a mds query? For example:

```
.opensearch(opensearch_dashboards_sample_data_logs, metric=avg:bytes, timefield=@timestamp).lines(show=true).points(show=true).yaxis(label="Average bytes"),
.opensearch(opensearch_dashboards_sample_data_logs, metric=avg:bytes, timefield=@timestamp, data_source_name="aos11").lines(show=true).points(show=true).yaxis(label="Average bytes")
```
```

Reply:
added